cfg/config.go

```diff
@@ -43,6 +43,9 @@ func setupCliFlags(version string, fs *flag.FlagSet, config *Configuration) {
 		"Timeout in seconds when collecting metrics from Fronius Symo. Should not be larger than the scrape interval.")
 	fs.Bool("symo.enable-power-flow", config.Symo.PowerFlowEnabled, "Enable/disable scraping of power flow data")
 	fs.Bool("symo.enable-archive", config.Symo.ArchiveEnabled, "Enable/disable scraping of archive data")
+	fs.Bool("symo.enable-smart-meter", config.Symo.SmartMeterEnabled, "Enable/disable scraping of smart meter data")
+	fs.Float64("symo.offset-consumed", config.Symo.OffsetConsumed, "Offset for consumed")
+	fs.Float64("symo.offset-produced", config.Symo.OffsetProduced, "Offset for produced")
```

Comment:
Please describe the flags better. What does "offset for consumed/produced" mean? What unit (if there is)?
If the values are only relevant for the smart meter endpoint, maybe make it `symo.smartmeter.offset-{consumed/produced}`?

Reply:
The offset is meant to make it possible to keep the metric in sync with the official electric meter. The unit is Wh.

Reply:
Ok, thanks. I'd propose to update the desc accordingly (better description welcome).
```suggestion
	fs.Float64("symo.offset-consumed", config.Symo.OffsetConsumed, "Offset in Wh added to consumed energy to keep in sync with official meter. Only used if smart-meter endpoint is enabled.")
	fs.Float64("symo.offset-produced", config.Symo.OffsetProduced, "Offset in Wh added to produced energy to keep in sync with official meter. Only used if smart-meter endpoint is enabled.")
```